cms/djangoapps/contentstore/views/utilities/captions.py

```diff
@@ -19,10 +19,11 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError, InsufficientSpecificationError
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys.edx.keys import UsageKey
-from xmodule.video_module.transcripts_utils import (
-                                    GetTranscriptsFromYouTubeException,
-                                    TranscriptsRequestValidationException,
-                                    download_youtube_subs)
+from xmodule.video_module.transcripts_utils import(
```

Comment:
You shouldn't edit this line; there should be a space after the `import` keyword.